### PR TITLE
[WWST-7290] Add Levition Dimmer Switch to zgibee-dimmer

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @SmartThingsCommunity/srpol-pe-team 
Could you review this?

### CODE REVIEW

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -40,6 +40,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05 ,1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: "FWST64Bulb50AU", deviceJoinName: "Aurora Light" //Aurora Dimmable Filament Vintage ST64
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05 ,1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: "FWG125Bulb50AU", deviceJoinName: "Aurora Light" //Aurora Dimmable Filament Vintage G125
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05 ,1000, FEDC", outClusters: "000A, 0019", manufacturer: "Aurora", model: "FWA60Bulb50AU", deviceJoinName: "Aurora Light" //Aurora Dimmable Filament Vintage GLS
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0301, 0B05", outClusters: "0019", manufacturer: "Leviton", model: "DG3HL", deviceJoinName: "Leviton Dimmer Switch" //Aurora Dimmable Filament Vintage GLS
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: @MGoralczykS Please, could You move this fingerprint to the section with existing Leviton fingerprints?

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy (hunk 1 above)
Comment (MGoralczykS) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: My bad, I'll fix it.

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 2]
```diff
@@ -82,6 +82,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0003, 0006, 0008, 0019, 0406", manufacturer: "Leviton", model: "DL3HL", deviceJoinName: "Leviton Dimmer Switch", ocfDeviceType: "oic.d.switch" //Leviton Lumina RF Plug-In Dimmer
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0003, 0006, 0008, 0019, 0406", manufacturer: "Leviton", model: "DL1KD", deviceJoinName: "Leviton Dimmer Switch", ocfDeviceType: "oic.d.switch" //Leviton Lumina RF Dimmer Switch
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0003, 0006, 0008, 0019, 0406", manufacturer: "Leviton", model: "ZSD07", deviceJoinName: "Leviton Dimmer Switch", ocfDeviceType: "oic.d.switch" //Leviton Lumina RF 0-10V Dimming Wall Switch
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0301, 0B05", outClusters: "0019", manufacturer: "Leviton", model: "DG3HL", deviceJoinName: "Leviton Dimmer Switch" //Aurora Dimmable Filament Vintage GLS
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: @MGoralczykS You might also need to update the comment after the finger print (and mention right device, now it's Aurora).

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy (hunk 2 above)
Comment (MGoralczykS) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Sorry, fixed

**Review by KKlimczukS — APPROVED** [review]

**Pull request merged**